packages/api/typedoc.js

```diff
@@ -6,7 +6,7 @@ module.exports = {
   exclude: [
     '**/node_modules/**/*',
     '**/errors/**/*',
-    '**/shared/**/*',
```

Comment:
This causes some effects on the generated documentation:

For instance, this class has been exported but the user will not receive it.

![Screenshot 2024-01-24 at 11 06 36](https://github.com/mymoid/mymoid-js/assets/118998693/51e15a70-f00f-4037-9244-cad64cfe8463)

Reply:
@mmarine-mymoid 

Try adding `'**/shared/endpoints-base*'`